config/runtime/tests/v4l2-decoder-conformance-chromeos.jinja2

```diff
@@ -0,0 +1,59 @@
+{%- include 'chromeos/base.jinja2' %}
+
+{%- if testsuite -%}
+    {% set testsuite_arg = "-ts " + testsuite %}
+{%- endif -%}
+{%- if decoders -%}
+    {%- set decoders_arg = "-d " + decoders|join(" ") -%}
+{%- endif -%}
+{%- if videodec_timeout -%}
+    {% set videodec_timeout_arg = "-t " + videodec_timeout|string %}
+{%- endif -%}
+{%- if videodec_parallel_jobs -%}
+    {% set videodec_parallel_jobs_arg = "-j " + videodec_parallel_jobs|string %}
+{%- endif -%}
+
+- test:
+    namespace: chromeos
+    timeout:
+      minutes: {{ job_timeout }}
+    docker:
+      image: kernelci/cros-tast
+      wait:
+        device: true
+    results:
+      location: /home/cros/lava
+    definitions:
+    - from: inline
+      name: {{ node.name }}
+      path: inline/{{ node.name }}.yaml
+      repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: cros-tast
+        run:
+          steps:
+            - cd /home/cros
+{%- if excluded_tests %}
+            - echo "# Disabled tests for KernelCI" > /tmp/excluded-tast-tests
+{%- for test in excluded_tests %}
+            - echo "-{{ test }}" >> /tmp/excluded-tast-tests
+{%- endfor %}
+{%- endif %}
+            - lava-test-set start setup
+            - >-
+              lava-test-case tast-tarball --shell
+              curl -s '{{ platform_config.params.tast_tarball }}'
+              \| tar xzvf -
+              && cp remote_test_runner /usr/bin/remote_test_runner
+              && mkdir -p /usr/libexec/tast/bundles/remote/
+              && cp cros /usr/libexec/tast/bundles/remote/
+            - for i in $(seq 1 60); do ping -c 1 -w 1 $(lava-target-ip) && break || sleep 1; done
+            - ping -c 1 -w 1 $(lava-target-ip) || lava-test-raise "cros-device-unreachable"
+            - >-
+              ./ssh_retry.sh
+              -o StrictHostKeyChecking=no
+              -o UserKnownHostsFile=/dev/null
+              -i /home/cros/.ssh/id_rsa
```

Comment:
From [changes to the pipeline config](https://github.com/kernelci/kernelci-pipeline/pull/705/files#diff-e055f21b05c11c155060b7dd48fa22812a879fada49f942fbf4368d9afbd5761R384) it looks like this TestJob Action should be ran on `bookworm-gst-fluster` rootfs.

When is the corresponding pubkey deployed to that image?

[I wasn't able to verify these changes](https://lava.collabora.dev/scheduler/job/15329364#bottom)

Reply:
Hi @pawiecz . Thank you for your review!
Yes, you are right, good catch. I removed the line which was wrongly using the debian rootfs. See https://github.com/kernelci/kernelci-pipeline/pull/705/files#diff-e055f21b05c11c155060b7dd48fa22812a879fada49f942fbf4368d9afbd5761